# Static — a quantitative teardown 🔬
### The idio-vol hedge with a Lo t-stat · the steady inversion · the MAX twin · micro-caps & survivorship

![Signal: None](https://img.shields.io/badge/Signal-None-c0392b?style=flat-square)
![Tradability: Mirage](https://img.shields.io/badge/Tradability-Mirage-c0392b?style=flat-square)
![Idio-vol puzzle on large caps?: Busted](https://img.shields.io/badge/Idio--vol_puzzle_on_large_caps%3F-Busted-8b949e?style=flat-square)

The deep companion to the [notebook for the curious](01_for_the_curious.ipynb). We show the low-vol puzzle inverts on tradable large caps.

> ⚠️ **Not investment advice.** 436 S&P 500 names + SPY, daily, 2000–2026 (Yahoo); survivorship-biased, large-cap. Sources in [`docs/references.md`](../docs/references.md).

In [1]:
import sys, os
sys.path.insert(0, os.path.abspath("../../.."))  # repo root (quantlab/)
sys.path.insert(0, os.path.abspath(".."))        # study package (static/)
%matplotlib inline
import matplotlib.pyplot as plt
plt.rcParams["figure.figsize"] = (10, 5.5); plt.rcParams["axes.grid"] = True
import numpy as np, pandas as pd
from static import data, strategy as st
daily, mkt = data.fetch_panel()                # cache-first
sig = st.idiovol_signal(daily, mkt)
textbook = st.cross_section_hedge(daily, sig, long_high=False)   # long low idio-vol


## Verdict, up front

| Axis | Stamp | Why |
|---|---|---|
| Signal | **None** | textbook trade −12.1%/yr, Sharpe −0.60, t −3.1 |
| Tradability | **Mirage** | negative gross, steady inversion |
| Large-cap puzzle? | **Busted** | lives in micro-caps |

> 💡 *In plain words:* the volatile names won; the puzzle is small-cap.

## 1 · The claim, steelmanned

- **H₁:** long-low/short-high idio-vol earns a positive premium.
- **H₂:** it's stable.
- **H₃:** it survives on tradable large caps.

## 2 · So what? — what rides on each

If H₁/H₃ hold, low-vol is a free lunch (more return, less risk). If they fail on large caps, the puzzle is a micro-cap, survivorship-fragile effect.

## 3 · How we'd know — the protocol

Rolling 1-factor idio-vol → long-low/short-high → Lo t-stat → decade split → compare to the MAX twin.

## 4 · The teardown

### 4.1 The textbook trade

In [2]:
import pandas as pd; display(pd.Series(st.stats(textbook)).round(3))

mean_ann     -0.121
sharpe       -0.605
tstat        -3.132
hit_rate      0.437
n           327.000
dtype: float64

> 💡 *In plain words:* negative and significant (t −3.1). **H₁ rejected** — inverted.

### 4.2 Steady, not a fluke

In [3]:
for lab,sl in [('2000-2012',textbook.loc[:'2012']),('2013-on',textbook.loc['2013':])]:
    print(f'{lab}: Sharpe {st.stats(sl)["sharpe"]:+.2f}')

2000-2012: Sharpe -0.65
2013-on: Sharpe -0.65


> 💡 *In plain words:* −0.65 in both halves — a structural inversion, not a regime blip. **H₂ rejected.**

### 4.3 Why — the high-risk axis, on large caps

Idio-vol and MAX ([53 Jackpot](../../53-jackpot/)) measure the same thing; both invert because (1) high idio-vol = high-risk growth, which *won* on large caps 2000–2026 (cf. [43 Free-Lunch](../../43-free-lunch/)); (2) survivorship keeps the high-vol *winners*; (3) the puzzle is documented in micro-caps (Hou-Xue-Zhang 2020). **H₃ rejected.**

## 5 · The verdict

H₁, H₂, H₃ all rejected → Signal `NONE`, Tradability `MIRAGE`, large-cap puzzle `BUSTED`.

## 6 · Could you trade it?

Not on liquid stocks — you'd short the leaders. The premium is real in small/illiquid names, where shorting is costly and survivorship worst.

## 7 · Going further

Forks: (a) point-in-time micro-cap universe; (b) value-weighting and NYSE breakpoints (Hou-Xue-Zhang); (c) idio-vol vs MAX head-to-head. Backlog: [`docs/pwb_strategies_inventory.md`](../../../docs/pwb_strategies_inventory.md).